java/ql/test/query-tests/security/CWE-927/SensitiveBroadcast.ql

```diff
@@ -0,0 +1,20 @@
+import java
+import semmle.code.java.security.AndroidSensitiveBroadcastQuery
+import TestUtilities.InlineExpectationsTest
```

Comment:
`java/ql/test/TestUtilities/InlineFlowTest.qll` can be imported instead of the boilerplate code.